### ISSUE TITLE
MirTop still broken (isomiR_perc)

### ISSUE THREAD

**Issue by apeltzer** [ewels/MultiQC#1719]
### Description of bug

@ErikDanielsson  - your latest bugfix fixed half of the low coverage issues we saw :-( Can we have such a defaulting to zero for all cases where there is a float division done in MultiQC? That would permanently fix it then... 

https://github.com/ewels/MultiQC/issues/1712

![image](https://user-images.githubusercontent.com/2359510/177099931-a8e20656-0831-4b51-9856-63346e572405.png)

### MultiQC Error log

```console
|         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 420/420
|    custom_content | nf-core-smrnaseq-summary: Found 1 sample (html)
|    custom_content | software_versions: Found 1 sample (html)
╭───────────────── Oops! The 'mirtop' MultiQC module broke... ─────────────────╮
│ Please copy this log and report it at                                        │
│ https://github.com/ewels/MultiQC/issues                                      │
│ Please attach a file that triggers the error. The last file found was:       │
│ ./93/full_mirtop_stats.log                                                   │
│                                                                              │
│ Traceback (most recent call last):                                           │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/multiqc.py", line 65 │
│     output = mod()                                                           │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/mirtop/mirto │
│     self.parse_mirtop_report(f)                                              │
│   File "/usr/local/lib/python3.10/site-packages/multiqc/modules/mirtop/mirto │
│     parsed_data["isomiR_perc"] = (parsed_data["isomiR_sum"] / parsed_data["r │
│ ZeroDivisionError: float division by zero                                    │
│                                                                              │
╰──────────────────────────────────────────────────────────────────────────────╯
|          samtools | Found 158 stats reports
|            fastqc | Found 79 reports
|           multiqc | Compressing plot data
|           multiqc | Report      : multiqc_report.html
|           multiqc | Data        : multiqc_data
|           multiqc | Plots       : multiqc_plots
|           multiqc | MultiQC complete
|           multiqc | 1 flat-image plot used in the report due to large sample numbers
|           multiqc | To force interactive plots, use the '--interactive' flag.
See the documentation.
```

**Comment by ewels** [comment]
@apeltzer is this will the example data in https://github.com/ewels/MultiQC_TestData/pull/227 ? If not, please could you provide some more? 😀

**Comment by apeltzer** [comment]
https://github.com/ewels/MultiQC_TestData/pull/234

**Comment by apeltzer** [comment]
Would be great if we can trigger another pre-release of MultiQC once @ErikDanielsson 's bugfix is in ;-) This still breaks smrnaseq :-(

**Comment by ErikDanielsson** [comment]
This should be fixed in #1723 @apeltzer.
